### DISCUSSION TITLE
API scripting issues at /providers/proxy

### DISCUSSION THREAD

**Discussion by kmcgreg5** [goauthentik/authentik#4616] [Q&A] [answered]
I'm unsure if this is a bug or not or where to add it if so.

Sending a post request to /api/v3/providers/proxy/ returns a 200 response and not the expected 201, 400, or 403 as shown in the API Browser and /api/v3/schema/ output.

The 200 response contains a list of the proxy providers as if returned from a get request, though I can't get the /providers/proxy get request parameters to affect the output. So a full list.

I'm executing this from python with the following code:
```
auth_flow_uuid = # retrieved from a successful get command to /core/applications/<slug>
params: dict={"name":"Some-Domain","authorization_flow":auth_flow_uuid, "external_host":"https://somedomain.mydomain.com"}
requests.Session().post(f'https://authentik.mydomain.com/api/v3/providers/proxy', json=params, headers={'Authorization':f'Bearer {my_token}'})
```

**Comment by BeryJu** [comment, marked as answer]
The API request is missing a trailing slash hence it gets redirected to a URL with a trailing slash, and turns into a GET request in the process
